Src/FluentAssertions.Json/JTokenAssertions.cs

```diff
@@ -469,10 +468,14 @@ public AndConstraint<JTokenAssertions> ContainSubtree(JToken subtree, string bec
 
         public string Format(JToken value, bool useLineBreaks = false)
         {
-            return new JTokenFormatter().Format(value, new FormattingContext
+            var output = new FormattedObjectGraph(100);
```

Comment:
The 100 is max number of lines, right?

Reply:
Yep. I've adding the argument name. If you ask me, it all feels a bit dirty, but I don't want to touch this right now.